src/openapi/openapi.json

```diff
@@ -1069,6 +1069,128 @@
         "x-router-controller": "cors"
       }
     },
+    "/api/data/updateForm": {
```

Comment:
Similar to previous PR comments, this endpoint should be `/api/data/form/{id}` to follow best practices: https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/#h2-2eb87cd076da0